assets/events.ts

```diff
@@ -303,11 +303,13 @@ export function toggleSidebar() {
     let div = document.querySelector(".DocsSidebar--sections .toggleSidebar")
     let btn = div.querySelector("button");
     btn.addEventListener("click", () => {
-      let classToggleList = ['.DocsSidebar', '.DocsToolbar', '.DocsFooter', '.DocsContent', '.DocsMarkdown', '.DocsSidebar--sections .toggleSidebar', '.breadcrumb'];
+      let classToggleList = ['.DocsSidebar', '.DocsToolbar', '.DocsContent', '.DocsMarkdown', '.DocsSidebar--sections .toggleSidebar', '.breadcrumb', '.DocsFooter'];
 
       classToggleList.forEach(function(querySelector){
         let item = document.querySelector(querySelector);
-        item.classList.toggle('collapsed');
+        if (item) {
+          item.classList.toggle('collapsed');
+        }
```

Comment:
I _believe_ you could use [optional chaining](https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/#optional-chaining) here, but I didn't test it:
```suggestion
        item?.classList.toggle('collapsed');
```

Reply:
I'll eval that for the future. Didn't know it was possible!